app/services/manage_braintree_donation.rb

```diff
@@ -0,0 +1,58 @@
+class ManageBraintreeDonation
```

Comment:
I was cribbing from `ManageAction` which is what enqueues actions, but if you'd like I could rename to something like `EnqueueBraintreeAction`?